canonical-web-and-design/ubuntu.com
# Cve rebase

## Done

- Rebased against master
- Fixed issues of `load_sample_script` with sqlite
- New migration created


## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- Populate releases data with `python webapp/security/fixtures/releases.py`
- Populate CVEs data with `python webapp/security/fixtures/load_sample_cve.py`
- Check http://localhost:8001
